README.rst

```diff
@@ -943,6 +944,46 @@ To use point ``STATICFILES_STORAGE`` to ``openwisp_utils.storage.CompressStaticF
 
     STATICFILES_STORAGE = 'openwisp_utils.storage.CompressStaticFilesStorage'
 
+Admin Theme utilities
+---------------------
+
+``openwisp_utils.admin_theme.email.send_email``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This function let you send customised mail using the template and logo that can be
```

Comment:
This functio allows sending email in both plain text and HTML version (using template and logo defined in the settings ... and ... respectively).